Ast/src/Lexer.cpp

```diff
@@ -89,7 +91,16 @@ Lexeme::Lexeme(const Location& location, Type type, const char* data, size_t siz
     , length(unsigned(size))
     , data(data)
 {
-    LUAU_ASSERT(type == RawString || type == QuotedString || type == Number || type == Comment || type == BlockComment);
+    LUAU_ASSERT( \
```

Comment:
No need for backslashes

Reply:
Whoops, this is holdover from another language I work in where you do